package.json

```diff
@@ -246,7 +243,6 @@
 		"@babel/eslint-parser": "^7.26.10",
 		"@babel/register": "^7.25.9",
 		"@babel/runtime": "^7.26.10",
-		"@bartekbp/typescript-checkstyle": "^5.0.0",
```

Comment:
Similarly, this and the `eslint-nibble` dependency below could be used in a developer's own terminal, but we don't have any documented workflows for it. I think these dependencies are the sort of thing where a developer could install them (optionally globally) in their own environment if they want to use them, rather than including them in the project by default.

Reply:
Yep, I actually am still using these through `npx` all the time. There's very little value in keeping them in - maybe if we have some very special persisting config through a config file.